### ISSUE TITLE
U3A -OUT supposed to connect to R14?

### ISSUE THREAD

**Issue by shanemmattner** [AlperenAkkuncu/PlainDAQ#3]
Should the `-OUT` be connected to R14 here?
![Screenshot from 2022-08-23 19-32-55](https://user-images.githubusercontent.com/22162612/186305814-facb0c6f-9c3e-4a3a-8fee-0ec857856385.png)

**Comment by AlperenAkkuncu** [comment]
Sorry for my late response @shanemmattner I am still getting used to github :)
Yes you are correct.
I will remove this schematic as it's not relevant right. I will upload the new version of the schematic files, once I launch the project on crowduspply.

In fact, I will share the whole KiCAD project :D

**Comment by shanemmattner** [comment]
No problem, <24 hours response time on Github is lightning quick haha.

I look forward to seeing the full KiCAD project.  A brief explanation of the circuits (ie U2A and U3A) or links to documents you used to make the circuits would be great as well.

**Comment by AlperenAkkuncu** [comment]
Thanks for the kind words :)
@shanemmattner are you asking the function of the FDA(fully differential amplifier?

**Comment by shanemmattner** [comment]
Yes, the FDA is a good example.  I didn't know the name of the circuit so I did not understand how it works.  I've created [this issue](https://github.com/AlperenAkkuncu/PlainDAQ/issues/4) for adding circuit theory documentation.  Let me know if I can help
